docs-gb/examples/k8s-examples.md

```diff
@@ -56,7 +56,7 @@ model.mlops.seldon.io/iris created
 kubectl wait --for condition=ready --timeout=300s model --all -n ${NAMESPACE}
 ```
 
-```
+```bash
```

Comment:
Same comment regarding output/bash for "condition met", "deleted", etc. If the formatting is ok then we can leave them as bash

Reply:
fixed